docs/en-US/component/tour.md

```diff
@@ -100,7 +100,7 @@ tour-step component configuration with the same name has higher priority
 
 | Property                 | Description                                                                                                  | Type                                                                                                                                                                        | Default   |
 | ------------------------ | ------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- |
-| target                   | get the element the guide card points to. Empty makes it show in center of screen                            | `HTMLElement`                                                                                                                                                               | -         |
+| target                   | get the element the guide card points to. Empty makes it show in center of screen                            | `HTMLElement` \| `string` \| `() => HTMLElement`                                                                                                                            | -         |
```

Comment:
Please add a describe the string type is selectors of document.querySelector

Reply:
done